suite-common/message-system/schema/config.schema.v1.json

```diff
@@ -253,6 +253,9 @@
                             "content": {
                                 "$ref": "#/definitions/localization"
                             },
+                            "title": {
```

Comment:
What would be the usage of title? Would you need both title and content in one message?

Reply:
Yes, exactly. Title, content and cta

Reply:
![Uploading IMG_9338.jpeg…]()

Reply:
![IMG_9338](https://github.com/trezor/trezor-suite/assets/2011829/d4a784be-e3a8-492f-8d06-2851e9096827)

Reply:
ok, please use another name than `title` because it's used as a keyword in the schema definition, so it can be confusing. What about `headline`?

Reply:
031db693b